docs/test.js

```diff
@@ -2074,13 +2074,14 @@ describe(".on(event, fn)", function () {
         done();
       }
     });
-    u('.cb-wrapper').on('change', '.cb', function () {
+    base.find('.cb-wrapper').on('change', '.cb', function () {
```

Comment:
Why is this change needed?

Reply:
This must've propagated from on/test.js as part of the build.